plugins/registry.js

```diff
@@ -35,32 +35,32 @@ export default class PluginRegistry {
 
     // Register a component at the root of the channel view of the app.
     // Accepts a React component. Returns a unique identifier.
-    registerRootComponent = (component) => {
+    registerRootComponent(component) {
```

Comment:
Binding the functions using arrow functions wasn't necessary, since we weren't using these callbacks out of context (e.g. as an event handler). This change allows me to use esprima to parse out this file and generate automated documentation.